src/modules/endpoint/endpoints/feedback/categoriesFeedback.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { apiUrl } from '../../constants'
+import ParamMissingError from '../../errors/ParamMissingError'
+import FeedbackEndpointBuilder from '../../FeedbackEndpointBuilder'
+
+const CATEGORIES_FEEDBACK_ENDPOINT_NAME = 'categoriesFeedback'
+const POSITIVE_RATING_STRING = 'up'
+const NEGATIVE_RATING_STRING = 'down'
+
+export default new FeedbackEndpointBuilder(CATEGORIES_FEEDBACK_ENDPOINT_NAME)
+  .withParamsToUrlMapper((params): string => {
+    if (!params.city) {
+      throw new ParamMissingError(CATEGORIES_FEEDBACK_ENDPOINT_NAME, 'city')
+    }
+    if (!params.language) {
+      throw new ParamMissingError(CATEGORIES_FEEDBACK_ENDPOINT_NAME, 'language')
+    }
+    return `${apiUrl}/${params.city}/${params.language}/wp-json/extensions/v3/feedback`
+  })
+  .withParamsToBodyMapper((params): FormData => {
```

Comment:
I think we should introduce a proper type here which has a method to convert to a FormData. This would make the code easier to debug. Not sure what the internals of a FormData are.